.travis.yml

```diff
@@ -5,17 +5,17 @@ addons:
     - xorg-dev
 language: go
 go:
-  - 1.11.x
-  - 1.10.x
-  - tip
+  - 1.x
+  - 1.12.x
+  - master
 matrix:
   allow_failures:
-    - go: tip
+    - go: master
   fast_finish: true
 install:
   - # Do nothing. This is needed to prevent default install action "go get -t -v ./..." from happening here (we want it to happen inside script step).
 script:
-  - go get -t -v ./v3.3/...
-  - diff -u <(echo -n) <(gofmt -d -s .)
-  - go tool vet .
-  - go test -v -race ./v3.3/...
+  - go get -t -v ./v3.3/... ./v3.2/...
+  - diff -n <(echo -n) <(gofmt -d -s .)
+  - go vet ./v3.3/... ./v3.2/...
+  - go test -v -race ./v3.3/... ./v3.2/...
```

Comment:
Hmm, is go test doing anything?

https://travis-ci.org/go-gl/glfw/jobs/651250770#L656-L658

```
$ go test -v -race ./v3.3/... ./v3.2/...
?   	github.com/go-gl/glfw/v3.3/glfw	[no test files]
?   	github.com/go-gl/glfw/v3.2/glfw	[no test files]
```

Reply:
Yes, it is still testing that the package builds successfully.

We don't have any tests, hence "[no test files]". If we add some, they'll be run.